src/main/java/com/palantir/docker/proxy/DockerProxySelector.java

```diff
@@ -45,7 +44,9 @@ public DockerProxySelector(Cluster containers, DockerContainerInfo containerInfo
 
     @Override
     public void connectFailed(URI uri, SocketAddress sa, IOException ioe) {
-        Preconditions.checkArgument(uri != null && sa != null && ioe != null, "Invalid connectFailed call");
+        com.palantir.logsafe.Preconditions.checkArgument(
```

Comment:
Can you convert this to an import instead of splitting it over multiple lines?

Reply:
eurgh, yeah, I thought it was doing the thing where it did a fq import if there were invocations of both `Preconditions` but sure okay.